packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -201,6 +201,21 @@ export class StripeHelper {
     return stripeCustomer;
   }
 
+  /**
+   * Insert a local db record for a customer that already exist on Stripe.
+   */
+  async createLocalCustomer(uid: string, stripeCustomer: Stripe.Customer) {
+    // There isn't a nice way to write INSERT...WHERE NOT EXIST in
```

Comment:
This solution doesn't work?
https://github.com/knex/knex/issues/1322#issuecomment-639618321

Reply:
I'm sure dropping down into `knex.raw` would work.

Reply:
Nothing wrong with using `knex.raw`, the intention with knex/objection was to get typing and a dynamic query builder when needed. If the query is static, I'm all in favor of using `knex.raw` so that its easier to maintain the queries without having to learn knex/objection query syntax in addition to SQL.

Reply:
Updated `createAccountCustomer` to use INSERT ... WHERE NOT EXISTS.